elections/steering/2023/README.md

```diff
@@ -64,18 +64,18 @@ previous [governance meeting video] which led to this whole process.
 
 Please refer to the [Steering Committee Election Charter] for [Eligibility for candidacy]
 
-Eligibility for voting in {{YEAR}} is defined as:
+Eligibility for voting in 2023 is defined as:
 
-* People who had at least {{NUM}} contributions to the Kubernetes project over
-  the past year, according to a snapshot taken {{DATE OF SNAPSHOT}} of the data driving
+* People who had at least 50 contributions to the Kubernetes project over
+  the past year, according to a snapshot taken 2022-08-04 of the data driving
```

Comment:
I think the year might be wrong here

Reply:
Thanks. Upon review, I found that there were a number of additional places I needed to correct the year.